pages/linux/rtorrent.md

```diff
@@ -0,0 +1,19 @@
+# rtorrent
+
+> Download torrents over the command line.
+
+- Add a torrent file or magnet to be downloaded:
+
+`rtorrent {{torrent or magnet}}`
```

Comment:
torrent_or_magnet_file

Reply:
It would be a torrent file or magnet link in that case.. should I change the text on line 7?

Reply:
Ah you are right. Let's keep it torrent_or_magnet then.